invenio_rdm_pure/source/rdm/marc21_record.py

```diff
@@ -46,15 +46,20 @@ def __init__(self, code: str = "", value: str = ""):
 class Marc21Record(object):
     """MARC21 Record class to facilitate storage of records in MARC21 format."""
 
-    def __init__(self, leader: str = ""):
+    def __init__(
+        self, leader: str = "97628dxefp 201109bOq4500"
```

Comment:
what are those magic numbers?

Reply:
The schema doesn't allow an empty leader, so I added a value, which conforms with the schema. Could you please give me more information how the leader should be handled in imports?

Reply:
The default value has been moved to a constant value.